.github/integration-test/basic-auth/docker-compose.yml

```diff
@@ -19,11 +19,23 @@ services:
     image: fhir-data-evaluator
     environment:
       CONVERT_TO_CSV: ${FDE_CONVERT_TO_CSV:-true}
-      FHIR_SERVER: "http://proxy:8080/fhir"
-      FHIR_USER: "test"
-      FHIR_PASSWORD: "bar"
+      FHIR_SOURCE_SERVER: "http://proxy:8080/fhir"
+      FHIR_SOURCE_USER: "test"
+      FHIR_SOURCE_PASSWORD: "bar"
+      FHIR_DESTINATION_SERVER: "http://proxy:8080/fhir"
+      FHIR_DESTINATION_USER: "test"
+      FHIR_DESTINATION_PASSWORD: "bar"
+      SEND_REPORT_TO_SERVER: ${FDE_SEND_REPORT_TO_SERVER:-false}
```

Comment:
I would rename that variable to `REPORT_TARGET` with values `source-server` and `report-server`.

Reply:
So then the report should be sent to the source-server in case it is set to `source-server`, and to the report server if set to `report-server` and if not set, it won't be sent?

Reply:
I would assume that `source-server` is the default. So it's stored (send) to the source server if nothing is specified. Not storing the report make no sense - or doesn't it?

Reply:
by default it is stored on disk at the output directory, which is mouted as a volume

Reply:
Ok then I would add a third option `disk` which is the default. You can also omit `source-server` if that makes no sense or is too expensive to implement.

Reply:
should i then not save it to the disk at all if it is not set to `disk`?